extensions/dicom-seg/src/components/SegmentItem.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { TableListItem, Icon } from '@ohif/ui';
+
+const ColouredCircle = ({ color }) => {
```

Comment:
~ColouredCircle~ --> `ColorCircle` or `ColoredCircle`
@JamesAPetts 👀

Reply:
That was me, although fits as a tribute to james...

Reply:
Ah. I was definitely being cheeky here, haha.